client2/.eslintrc.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
         "max-len": [
             "error",
             {
-                code: 100,
+                code: 150,
```

Comment:
@Arcanorum I adjusted the max-len since some of the previous codes exceed 100 characters per line.

Do you think that we should keep this? or should I revert it back in the upcoming commits?

Reply:
I think 100 is enough. If there is some lines you think really need to be longer than this, you can disable the rule for that line by putting `// eslint-disable-next-line max-len` on the line above it.

Reply:
Alright I reverted it back to 100
https://github.com/Arcanorum/dungeonz/pull/99/commits/b011f5bf33c3b36269c5e7f931a342eb93e218d7